cmake/DefaultTargetProperties.cmake

```diff
@@ -1,5 +1,10 @@
 # SPDX-License-Identifier: Apache-2.0
 # Copyright 2013-2022 The Foundry Visionmongers Ltd
+include_guard(GLOBAL)
```

Comment:
[comment/no action] I didn't realise that CMake had include_guards.  I can see why it needs them, but I think it's ugly that such a high level language requires them.